modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -452,7 +452,8 @@ cv::GStreamingCompiled cv::gimpl::GCompiler::produceStreamingCompiled(GPtr &&pg)
         outMetas = GModel::ConstGraph(*pg).metadata().get<OutputMeta>().outMeta;
     }
 
-    std::unique_ptr<GStreamingExecutor> pE(new GStreamingExecutor(std::move(pg)));
+    std::unique_ptr<GStreamingExecutor> pE(new GStreamingExecutor(std::move(pg),
+                                                                  m_args));
```

Comment:
So why do we pass `m_args` to the Streaming Executor but don't pass those to the `GExecutor` in the `produceCompiled`?

Why do we need to pass *compile arguments* to *executor* at all?

Reply:
Discussed in IM, comment about compile arguments added before GStreamingExecutor constructor definition